python-api/gstswitch/server.py

```diff
@@ -40,7 +40,7 @@ def __init__(
             video_port=3000,
             audio_port=4000,
             control_port=5000,
-            record_file='record.data'):
+            record_file='record-%Y-%m-%d_%H%M%S.data'):
```

Comment:
This should be left empty by default I think.

`record_file=True` means `-r`, `record_file='record-%Y-%m-%d'` should be the same as `--record="record-%Y-%m-%d"`